pkg/sdn/plugin/ovscontroller.go

```diff
@@ -254,16 +279,66 @@ func (oc *ovsController) SetPodBandwidth(hostVeth string, ingressBPS, egressBPS
 	return nil
 }
 
-func (oc *ovsController) UpdatePod(hostVeth, podIP, podMAC string, vnid uint32) error {
-	ofport, err := oc.ensureOvsPort(hostVeth)
+func getPodDetailsByContainerID(flows []string, containerID string) (int, string, string, string, error) {
+	note, err := getPodNote(containerID)
+	if err != nil {
+		return 0, "", "", "", err
+	}
+
+	// Find the table=20 flow with the given note and extract the podIP, ofport, and MAC from them
+	for _, flow := range flows {
+		parsed, err := ovs.ParseFlow(ovs.ParseForAdd, flow)
```

Comment:
You're not doing an "add" here. There probably needs to be another ParseCmd value.